components/update_client/BUILD.gn

```diff
@@ -22,3 +22,13 @@ buildflag_header("buildflags") {
     "UPDATER_PROD_ENDPOINT=\"$updater_prod_endpoint\"",
   ]
 }
+
+if (is_mac) {
+  source_set("update_client") {
```

Comment:
nit: I think this target name might be a bit confusing. Since it's only features I think it would be better to call the target `features` and there are lots of examples of this in upstream

Reply:
I renamed the target to `features` in 0f206a02362c70104a0e0443e942cbbcd4ff6a16.

Reply:
I rebased and `0f2...` is now c2fb5769b449d5377753ef4348adcd9d0dc56294.